network/testing/testing.go

```diff
@@ -0,0 +1,347 @@
+package testing
```

Comment:
Fixed 👍 

[1b09821](https://github.com/0xPolygon/polygon-edge/pull/343/commits/1b09821386f05887470dfe532f459632012b725c)

Reply:
Actually this can't be a `_test.go` file because it needs to be imported elsewhere.

I've reverted it here:

[47460e1](https://github.com/0xPolygon/polygon-edge/pull/343/commits/47460e1008876753ddd3259a9d2d52e0605c6573)